test/second_q/drivers/gaussiand/test_driver_gaussian_forces.py

```diff
@@ -92,7 +119,34 @@ class TestDriverGaussianForces(QiskitNatureTestCase):
         [2.2973803125, 3, 3, 3, 3],
     ]
 
-    def _get_expected_values(self):
+    _A03_REV_PBE_EXPECTED = [
+        [353.5831025, 2, 2],
+        [-353.5831025, -2, -2],
+        [644.7579625, 1, 1],
+        [-644.7579625, -1, -1],
+        [95.344445, 4, 4],
+        [-95.344445, -4, -4],
+        [95.344445, 3, 3],
+        [-95.344445, -3, -3],
+        [-15.110404634225285, 2, 2, 2],
+        [0.409219375, 2, 2, 2, 2],
+        [1.5162932291666669, 1, 1, 1, 1],
+        [3.5002357291666666, 4, 4, 4, 4],
+        [3.5002358333333334, 3, 3, 3, 3],
+        [-85.62267978593646, 1, 1, 2],
+        [53.86795497291527, 4, 4, 2],
+        [53.86795497291527, 3, 3, 2],
+        [4.740964375, 1, 1, 2, 2],
+        [-5.103499375, 4, 4, 2, 2],
+        [-5.103499375, 3, 3, 2, 2],
+        [-12.17633125, 4, 4, 1, 1],
+        [-12.17633125, 3, 3, 1, 1],
+        [3.984481666666667, 4, 4, 4, 3],
+        [10.624813125, 4, 4, 3, 3],
+        [-3.984481666666667, 4, 3, 3, 3],
+    ]
```

Comment:
@woodsp-ibm I have added a functional test below which will run the usual test but using a different functional (specifically the `PBE` Exchange and `PBE` Correlation functional (see also [here](https://gaussian.com/dft/))).
Could you please run this test once with the A03 version which you have access to and update this list here to match the result that you get?